tasks/haproxy_generate_ssl.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
should have a preflight task to check an old cert and clean up the old certs before generating new certs.